FlowCrypt/Functionality/Services/Google User Service/GoogleAuthManager.swift

```diff
@@ -216,20 +212,12 @@ extension GoogleAuthManager {
 
     // save auth session to keychain
     private func saveAuth(state: OIDAuthState, for email: String) {
+        currentUserEmail = email // Save email variable so that OIDAuthStateChangeDelegate can access
```

Comment:
I think we can get `email` value from `OIDAuthState`, so no need to separate `email` param:
```swift
let authorization = GTMAppAuthFetcherAuthorization(authState: state)
GTMAppAuthFetcherAuthorization.save(authorization, toKeychainForName: Constants.index + authorization.userEmail)
```
What do you think, will it work here?

Reply:
Yeah, You are right.
I think it will work.